i18n/sync_tx_translations.js

```diff
@@ -23,19 +23,16 @@ if (!process.env.TX_TOKEN || process.argv.length !== 2) {
 const fs = require('fs');
 const path = require('path');
 const assert = require('assert');
-const transifex = require('transifex');
 const locales = require('scratch-l10n').default;
+const txPull = require('./transifex.js');
```

Comment:
Since `scratch-blocks` already depends on `scratch-l10n` it seems like, once LLK/scratch-l10n#175 is merged, this could be:
```suggestion
const {txPull} = require('scratch-l10n/lib/transifex.js');
```
...or you could re-export it from `scratch-l10n/src/index.js` and potentially merge this proposed line with the `locales` line above.

Anyway, either option would let you remove the duplicated code in this PR's `i18n/transifex.js`, and it looks like that might make it unnecessary to depend on the Transifex API or `download` packages directly as well.

Reply:
oh awesome! wasn't sure if there was a good way to import this stuff directly from scratch-l10n, I'd much rather do that. I'll change that once the l10n PR gets merged!